db/migrate/20250611083925_fix_stale_billing_entity_sequential_id_to_be_uniq.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+class FixStaleBillingEntitySequentialIdToBeUniq < ActiveRecord::Migration[8.0]
+  class Invoice < ApplicationRecord
+  end
+
+  class BillingEntity < ApplicationRecord
+  end
+
+  def up
+    # BillingEntities::ChangeInvoiceNumberingService -- if we switch from per_customer
+    # to per_billing_entity, we'll recalculate the billing_entity_sequential_id
+    BillingEntity.where(document_numbering: "per_customer").find_each do |billing_entity|
```

Comment:
PR description mention all invoices for billing entities with document numbering "per_customer" will be set as billing_entity_sequential_id = nil however, code only updates duplicates... is it expected?

Reply:
yep, initially i was thinking to update all invoices, but then I decided to not nullify data that is correct. Updating the PR description, thank you :)